stress/common/src/main/java/alluxio/stress/worker/WorkerBenchParameters.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.worker;
+
+import alluxio.stress.Parameters;
+
+import com.beust.jcommander.Parameter;
+
+/**
+ * Parameters used in the UFS I/O throughput test.
+ * */
+public class WorkerBenchParameters extends Parameters {
```

Comment:
I think this should be renamed to something like `UfsIOParameters`? The ony test that uses this class is a test called `UfsIOBench`.